osh.spec

```diff
@@ -289,7 +295,8 @@ pg_isready -h localhost && %{python3_sitelib}/osh/hub/manage.py migrate
 
 %files hub-conf-devel
 %defattr(644,root,apache)
-%config(noreplace) %{python3_sitelib}/osh/hub/settings_local.py
+%{python3_sitelib}/osh/hub/settings_local.py
+%config(noreplace) %{_sysconfdir}/osh/hub/settings_local.py
 %config(noreplace) %{python3_sitelib}/osh/hub/__pycache__/settings_local*.pyc
```

Comment:
We do not want to include the byte-compiled content of `settings_local.py` into the package because the corresponding Python code is moved to `/etc/osh/hub` and can be modified by the admin.